opm/parser/share/keywords/000_Eclipse100/O/OPERNUM

```diff
@@ -0,0 +1,5 @@
+{
+ "name" : "OPERNUM",
```

Comment:
please add this to the integer grid properties. (it can be manipulated with modifier keywords, right?)